torchvision/transforms/transforms.py

```diff
@@ -2014,3 +2014,51 @@ def forward(self, img):
 
     def __repr__(self):
         return self.__class__.__name__ + f"(p={self.p})"
+
+
+class ElasticDeformation(torch.nn.Module):
+    """Transform a tensor image with elastic deformations.
+    This transform does not support PIL Image.
+    Given alpha and sigma, it will generate displacement
+    vectors for all pixels based on random offsets. Alpha controls the strength
+    and sigma controls the smoothness of the displacements.
+    The displacements are added to an identity grid and the resulting grid is
+    used to grid_sample from the image.
+    Applications:
+        Randomly transforms the morphology of objects in images and produces a
+        see-through-water-like effect.
+    Args:
+        alpha: List [alpha H, alpha W] or (float), defines the magnitude of displacements.
+        sigma: List [sigma H, sigma W] or (float) defining the smoothness of displacements.
+        interpolation: str saying either NEAREST, BILINEAR or BICUBIC. Defining the interpolation used in grid_sample.
```

Comment:
interpolation arg should be similar to this one https://github.com/pytorch/vision/blob/fc52e86fa34413d35713ce264255a9f00902b2e2/torchvision/transforms/transforms.py#L748-L751